src/status_im2/contexts/syncing/scan_sync_code/view.cljs

```diff
@@ -199,7 +215,14 @@
                                                         :icon-color colors/danger-50
                                                         :override-theme :light
                                                         :text (i18n/label
-                                                               :t/camera-permission-denied)}])}]))]
+                                                               :t/camera-permission-denied)}])}]))
+        perform-preflight-check   (fn []
```

Comment:
consider moving this whole function to a separate defn and leave the comment with it there. Also perhaps this atom should be passed as a parameter to the function.